R/map_peptides_on_structure.R

```diff
@@ -1,34 +1,38 @@
 #' Maps peptides onto a PDB structure
 #'
 #' Peptides are mapped onto PDB structures based on their positions. This is accomplished by replacing the B-factor information in the structure file with values that allow
-#' highlighting peptides when the structure is coloured by B-factor. In addition to only highlighting peptides, a continuous variable such as fold changes associated with peptides
-#' can be mapped onto the structure as a colour gradient.
+#' highlighting of peptides, protein regions and amino acids when the structure is coloured by B-factor. In addition to only highlighting peptides,
+#' a continuous variable such as fold changes associated with peptides can be mapped onto the structure as a colour gradient.
```

Comment:
these changes don't have to be associated with peptides only, they can also be associated with regions and/or amino acids.